packages/quantic/force-app/coveo-v2-token/main/default/classes/CoveoV2TokenProvider.cls

```diff
@@ -0,0 +1,16 @@
+global with sharing class CoveoV2TokenProvider implements ITokenProvider {
```

Comment:
Not need to put "CoveoV2", call it CoveoTokenProvider instead ?

Reply:
- [ ] Rename CoveoV2TokenProvider to CoveoTokenProvider